packages/odyssey-react/src/components/Table/TableCell.module.scss

```
@@ -12,32 +12,32 @@
 
 .root {
   border-block-end: var(--BorderWidth) var(--BorderStyle) var(--BorderColor);
-  max-width: var(--MaxLineLength);
-  padding-block: var(--SpacePaddingBlock);
-  padding-inline: var(--SpacePaddingInline);
+  max-width: var(--MaxWidth);
+  padding-block: var(--PaddingBlock);
+  padding-inline: var(--PaddingInline);
   text-align: start;
   vertical-align: baseline;
   overflow-wrap: break-word;
 }
 
 .headerCell {
-  color: var(--ColorTextHeader);
-  font-weight: var(--FontWeightHeader);
+  color: var(--HeaderTextColor);
+  font-weight: var(--HeaderFontWeight);
 }
 
 .headerCellCol {
-  border-block-start: var(--BorderWidthHeaderCol) var(--BorderStyleHeaderCol)
-    var(--BorderColorHeaderCol);
+  border-block-start: var(--HeaderCellColBorderWidth)
```

```suggestion
  border-block-start-width: var(--HeaderCellColumnBorderWidth);
  border-block-start-style: var(--HeaderCellColumnBorderStyle);
  border-block-start-color: var(--HeaderCellColumnBorderColor);
```

also will abbreviations like `Col` be problematic here? I think a long form name may be easier to reason about?

---

addressed in edef7d7